ginko/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "ginko",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0"
+  },
+  "devDependencies": {
+    "react-scripts": "1.0.14"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  }
+}
```

Comment:
We should have an option to grab everything from CDNJS as well where possible~
I think we can do that in a followup ticket.